CoEpi/ui/symptomReport/FollowUp/Cough/CoughViewController1.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+class CoughViewController1: UIViewController {
+    private let viewModel: CoughViewModel1
+    
+    //Button Actions
```

Comment:
Redundant comments (also in other places). It's clear that these are button actions.

Reply:
removed